cmd/podman/shared/pod.go

```diff
@@ -60,3 +61,35 @@ func GetPodStatus(pod *libpod.Pod) (string, error) {
 	}
 	return created, nil
 }
+
+// GetNamespaceOptions transforms a slice of kernel namespaces
+// into a slice of pod create options. Currently, not all
+// kernel namespaces are supported, and they will be returned in an error
+func GetNamespaceOptions(ns []string) ([]libpod.PodCreateOption, error) {
+	var options []libpod.PodCreateOption
+	var erroredOptions []libpod.PodCreateOption
+	for _, toShare := range ns {
+		switch toShare {
+		case "net":
+			options = append(options, libpod.WithPodNet())
+		case "mnt":
+			//options = append(options, libpod.WithPodMNT())
```

Comment:
I'd drop this commented out piece of code, or leave a comment as to when/how it should be turned back on.